mock-relying-party-ui-telecom/src/components/theme.json

```diff
@@ -0,0 +1,9 @@
+{
+  "colors": {
+    "backgroundColor": "#F7FBFC"
```

Comment:
Add 
1. button colors. Primary and Secondary
    Primary can be assigned to button background
    Secondary can be assigned to border and hover
2. text primary and secondary
3. Font
etc